### PR TITLE
RHPAM-2484 - aligning tomcat-dbcp version

### PR DESCRIPTION
**Pull request by mareknovotny**
https://issues.redhat.com/browse/RHPAM-2484

### CODE REVIEW

**Comment by mareknovotny** [comment]
Jenkins execute full downstream build

**Comment by mareknovotny** [comment]
Jenkins execute full downstream build

**Comment by mareknovotny** [comment]
Jenkins execute full downstream build

File: narayana-integration-bom/pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
     <!-- narayana dependencies for tomcat and kie server -->
     <version.org.jboss.narayana.tomcat>5.9.0.Final</version.org.jboss.narayana.tomcat>
     <!-- DBCP connection pooling for Narayana -->
-    <version.org.apache.tomcat.tomcat-dbcp>9.0.13</version.org.apache.tomcat.tomcat-dbcp>
+    <version.org.apache.tomcat.tomcat-dbcp>9.0.21</version.org.apache.tomcat.tomcat-dbcp>
```
Comment (rsynek) [inline narayana-integration-bom/pom.xml]: potentially we should update, test and release also narayana-tomcat integration library, as it depends on tomcat too:
https://github.com/web-servers/narayana-tomcat/blob/master/pom.xml#L38

**Pull request merged**